slu/slu/utils/preprocessing.py

```diff
@@ -58,7 +58,7 @@ def make_reftime_column_uniform(data_frame: pd.DataFrame) -> pd.DataFrame:
 
     return data_frame
 
-def make_data_column_uniform(data_frame: pd.DataFrame) -> pd.DataFrame:
+def make_data_column_uniform(data_frame: pd.DataFrame) -> None:
```

Comment:
Why does the type hint need a change?

Reply:
This was a mistake, return type hint shouldn't be removed. Fixed this in a new commit.